packages/typescript-library-generator/README.md

```diff
@@ -0,0 +1,76 @@
+# @pagopa/typescript-library-generator
+
+A PlopJS generator for creating TypeScript libraries in the DX monorepo that follows established conventions and best practices and uses tsup for building.
+
+## Features
+
+- 🚀 **TypeScript Library Generator**: Creates new TypeScript libraries with all necessary configuration
+- 📦 **Turborepo Integration**: Follows monorepo structure and conventions
+- 🔧 **Pre-configured Tools**: Includes ESLint, TypeScript, and other essential tools
+- 📝 **Template-based**: Uses Handlebars templates for consistent library structure
+- 🎯 **Workspace Aware**: Automatically detects and works within the monorepo structure
+
+## Installation
+
+From non-DX repositories:
+
+```bash
+pnpm add -D @pagopa/typescript-library-generator
+```
+
+## Usage
+
+When using in the DX monorepo, you can run the generator directly from the root directory:
+
+```bash
+pnpm run turbo gen typescript-library-generator
```

Comment:
ditto

I think the right command should be `pnpm exec turbo gen typescript-library-package`.
If you prompt `typescript-library-generator`, the CLI is asking you to select the generator to use; options are `typescript-package` and `typescript-library-package`

Reply:
it's ok `pnpm turbo gen` as well